game/scripts/vscripts/Sections/Section1.ts

```diff
@@ -0,0 +1,75 @@
+import * as tg from "../TutorialGraph/index"
+import * as tut from "../Tutorial/Core"
+import { getPlayerHero } from "../util"
+
+let graph: tg.TutorialStep | undefined = undefined
+let graphContext: tg.TutorialContext
+
+enum Section01EntityKeys {
+    CustomMudGolem = "npc_custom_mud_golem",
+    MudGolemTop = "mud_golem_top",
+    MudGolemBot = "mud_golem_bot"
+}
+
+const onStart = (complete: () => void) => {
+    print("Starting", "Section01");
+    CustomGameEventManager.Send_ServerToAllClients("section_started", { section: SectionName.Section01 })
```

Comment:
Might be nice to rename Section01 to a name that describes what this section of the tutorial is explaining. (I haven't read the script but I guess this is the part of the tutorial that explains how golem society works?)

Reply:
Yep, renamed to Section Opening